app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
```

Comment:
- [ ] There should be another association here between user and likes since a user can have one or many likes.